omnibus/config/software/postgresql13.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "postgresql13"
```

Comment:
Would it be possible for us to update https://github.com/chef/omnibus-software/blob/main/config/software/postgresql.rb and use that instead so we're not maintaining slightly different PG definitions in omnibus software, server, and supermarket?

Reply:
@tas50 we thought of that but the challenge that we faced is that we need to have the binaries of PG stored in a specific folder that has been defined in this custom pg-13 software definition in supermarket. Hence a separate software definition.

Reply:
If there's a reason for it to be different this is fine